nautobot/core/events/__init__.py

```diff
@@ -12,6 +16,43 @@
 logger = logging.getLogger(__name__)
 
 
+def load_event_brokers(nautobt_event_brokers):
+    """Process plugins and log errors if they can't be loaded."""
+    for publisher_name, publisher in nautobt_event_brokers.items():
+        options = publisher.get("OPTIONS", {})
+        topics = publisher.get("TOPICS", {})
+        if not isinstance(topics, dict):
+            raise EventPublisherImproperlyConfigured(
+                f"Malformed Event Publisher Settings: Expected `TOPICS` to be a 'dict', instead a '{type(topics).__name__}' was provided"
+            )
+        include_topics = topics.get("INCLUDE", ["*"])
+        if not isinstance(include_topics, list):
+            raise EventPublisherImproperlyConfigured(
+                f"Malformed Event Publisher Settings: Expected `INCLUDE` to be a 'list', instead a '{type(include_topics).__name__}' was provided"
+            )
+        exclude_topics = topics.get("EXCLUDE", [])
+        if not isinstance(exclude_topics, list):
+            raise EventPublisherImproperlyConfigured(
+                f"Malformed Event Publisher Settings: Expected `EXCLUDE` to be a 'list', instead a '{type(exclude_topics).__name__}' was provided"
+            )
+        options.update({"include_topics": include_topics, "exclude_topics": exclude_topics})
+        try:
+            event_publisher_path = publisher["CLASS"]
+            module_path, class_name = event_publisher_path.rsplit(".", 1)
+            module = importlib.import_module(module_path)
+            event_publisher_class = getattr(module, class_name)
+            event_broker = event_publisher_class(**options)
```

Comment:
While you're doing type-checking, should we assert that `issubclass(event_publisher_class, EventBroker)`?